### DISCUSSION TITLE
want to reclaim github student pack from new github account.I dont have access to my previous id.

### DISCUSSION THREAD

**Discussion by sadman-ruet** [community/community#139900] [GitHub Education]
### Hi everyone,

i have claimed github student pack with my institutional mail but after that i lost access with my git hub account i manage to unlink my mail from that id but can not use the institutional mail for student pack any more as it is showing the mail has been use to claim student pack for previous id

**Comment by davevad93** [comment]
Hi @sadman-ruet ,

I don't think it's possible, because even if you have unlinked your academic email from the old account, the student benefits are still tied to them, so you can't reuse your academic email to apply with your new account.

This is what's stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student):

- ***You cannot transfer academic discounts from one account to another. If you have more than one account you want to apply the discount to, consider [merging](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) your personal accounts and [renaming](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username) the retained account if desired***
